sources/packages/api/src/services/user/user.service.ts

```diff
@@ -58,4 +58,29 @@ export class UserService extends DataModelService<User> {
   async getActiveUser(userName: string) {
     return this.repo.findOne({ userName: userName, isActive: true });
   }
+
+  /**
+   * Creates or updates Ministry user information.
+   * @param userName user name as it is on KeyCloak.
+   * @param email email received from identity provider.
+   * @param givenNames givenNames received from identity provider.
+   * @param lastName lastName received from identity provider.
+   * @returns created/updated user.
+   */
+  async syncAESTUser(
+    userName: string,
+    email: string,
+    givenNames: string,
```

Comment:
shouldn't it be `givenName` instead of `givenNames`

Reply:
Given names are all the names that are not family names, so it could be more than one.

Reply:
if there are more than one name, then should the type be a `list`? or am I missing something

Reply:
GivenNames is not something being introduced in the PR it is being used in many places.
The GivenNames is a property in the Keycloak token that is mapped to our sims.users.firstName column on db.
Does not matter if there is one or more, it will be mapped to the user firstName.